index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Roman Numeral Converter</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      text-align: center;
+      background: linear-gradient(90deg, white, green, black);
+      color: white;
+      margin: 0;
+      padding: 20px;
+      box-sizing: border-box;
+    }
+
+    h1 {
+      color: black;
+    }
+
+    label {
+      display: block;
+      font-size: 20px;
+      margin-bottom: 10px;
+    }
+
+    input[type="number"] {
+      width: 200px;
+      height: 30px;
+      font-size: 18px;
+      padding: 5px;
+    }
+
+    button {
+      background-color: green;
+      color: white;
+      font-size: 20px;
+      padding: 10px 20px;
+      border: none;
+      cursor: pointer;
+    }
+
+    button:hover {
+      background-color: darkgreen;
+    }
+
+    #output {
+      font-size: 24px;
+      margin-top: 20px;
+    }
+  </style>
```

Comment:
- You need to have your CSS code in a separate file like `styles.css` and add its link using `<link>` tag inside your `<head>` tag in your HTML file.